src/release-notify-action.js

```diff
@@ -24,6 +26,21 @@ async function runAction(token, staleDays, commitMessageLines) {
   - author:${latestRelease.author.login}
 `)
 
+  const closedNotifyIssue = await getLastClosedNotifyIssue(
+    token,
+    latestRelease.published_at,
+    staleDays
+  )
+
+  if (closedNotifyIssue) {
+    const issueNo = await createSnoozeIssue(
+      token,
+      unreleasedCommits,
+      latestRelease
+    )
+    logInfo(`Snooze issue has been created. Issue No. - ${issueNo}`)
+  }
+
```

Comment:
I'm not exactly sure this logic is correct and it feels almost too simple, but I haven't though about it thoroughly yet. let's write tests and they'll make it emerge more clearly